docker/nginx.conf

```diff
@@ -1,65 +0,0 @@
-daemon off;
```

Comment:
Ok, I will revert, but if we switch back to nginx in the future, this file will be located in a sub-folder (probably `/docker/web/nginx.conf`) to have a clear config.